src/App.js

```diff
@@ -18,7 +18,7 @@ const App = () => (
       <Header />
       <Navbar />
       <div className="app-wrapper-content">
-        <Route path="/profile" render={() => <ProfileContainer />} />
+        <Route path="/profile/:userId?" render={() => <ProfileContainer />} />
```

Comment:
Why do you have "?" In the route?
"?" belongs to the parameters of the string (query parameters)

Reply:
Turns out it does  rout relative

Reply:
Could you explain it?
Or provide a link with explanation

Reply:
if you specify without a sign?
Then our route profileID will watch only the link profileID
(/profile  = empty page)
If you add a ? at the end, then the profile link also displays the user's default page

Reply:
Oh, I get it, interesting ;)